pkg/visibility/server.go

```diff
@@ -88,7 +88,7 @@ func applyVisibilityServerOptions(config *genericapiserver.RecommendedConfig) er
 	o.Etcd = nil
 	o.SecureServing.BindPort = 8082
 	// The directory where TLS certs will be created
-	o.SecureServing.ServerCert.CertDirectory = "/tmp"
+	o.SecureServing.ServerCert.CertDirectory = "/visibility"
```

Comment:
```suggestion
	o.SecureServing.ServerCert.CertDirectory = certDir
```

could you define this as a constant?

```diff
var (
	setupLog = ctrl.Log.WithName("visibility-server")
	// Admission plugins that are enabled by default in the kubeapi server
	// but are not required for the visibility server.
	disabledPlugins = []string{
		validatingadmissionpolicy.PluginName,
		resourcequota.PluginName,
		validatingwebhook.PluginName,
		mutatingwebhook.PluginName,
	}
+       certDif = "/visibility"
)
```

Reply:
updated.